javascript/ql/lib/semmle/javascript/internal/ConceptsImports.qll

```diff
@@ -0,0 +1,6 @@
+/**
+ * This file contains imports required for the JavaScript version of `ConceptsShared.qll`.
+ * Since they are language-specific, they can't be placed directly in that file, as it is shared between languages.
+ */
+
+import semmle.javascript.dataflow.DataFlow
```

Comment:
```suggestion
import semmle.javascript.dataflow.DataFlow::DataFlow as DataFlow
```
? 

See https://github.com/github/codeql/pull/8477#issuecomment-1070911561

Reply:
Would you perfer it this way, or are you just discussing different solutions? (not 100% clear to me)

Reply:
Great, I mentioned this to @hmac, and we agreed to go with this suggestion, since it should JustWork™. If it turns out to cause problems, we can fix it when we need it.